app/controllers/api/v2/rh_cloud/inventory_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V2
+    module RhCloud
+      class InventoryController < ::Api::V2::BaseController
+        include ::Api::Version2
+        include InventoryUpload::ReportActions
+        include InventoryUpload::TaskActions
+
+        api :GET, "/organizations/:organization_id/rh_cloud/report", N_("Download latest report")
+        def download_file
+          filename, file = report_file(params[:organization_id])
+
+          send_file file, disposition: 'attachment', filename: filename
+        end
+
+        api :POST, "/organizations/:organization_id/rh_cloud/report", N_("Start report generation")
+        def generate_report
+          organization_id = params[:organization_id]
+
+          start_report_generation(organization_id)
+
+          render json: {
+            action_status: 'success',
+          }, status: :ok
```

Comment:
Would it make sense to return task id here for later reference?

Reply:
I want to change the interface from AJ to tasks. Once I do it, I will add some output.
If I add an output right now, I think it would be harder for the users to change the consumer